--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -3,7 +3,7 @@
 import * as nativeFs from "fs";
 
 //#region rolldown:runtime
-var __require = /* @__PURE__ */ createRequire(import.meta.url);
+var fdirReq = typeof createRequire !== "undefined" ? /* @__PURE__ */ createRequire(import.meta.url) : null;
 
 //#endregion
 //#region src/utils.ts
@@ -440,8 +440,10 @@
 let pm = null;
 /* c8 ignore next 6 */
 try {
- __require.resolve("picomatch");
- pm = __require("picomatch");
+ if (fdirReq) {
+  fdirReq.resolve("picomatch");
+  pm = fdirReq("picomatch");
+ }
 } catch {}
 var Builder = class {
  globCache = {};
